doom-modeline-segments.el

```diff
@@ -2934,6 +2935,21 @@ mouse-3: Restart preview"
        (propertize (format "Follow %d/%d" (- nwindows nfollowing) nwindows)
                    'face 'doom-modeline-buffer-minor-mode)))))
 
+(doom-modeline-def-segment date
+  "date"
+  (when (bound-and-true-p display-time-mode)
+    (propertize
+     (concat
+      (when doom-modeline-icon
```

Comment:
It's not necessary to check `doom-modeline-icon` here since it's checked in `doom-modeline-icon` function.

Reply:
updated